tests/phpunit/elementor/modules/checklist/classes/test-kit-installation.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Elementor\Tests\Phpunit\Elementor\Modules\Checklist\Classes;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Test_Kit_Installation extends Step_Test_Base {
+	public function tearDown(): void {
+		parent::tearDown();
+	}
+
+	public function test__default_kit_installed() {
+		parent::setUp();
+
+		$this->set_wordpress_adapter_mock( [], [
+			'set_user_preferences' => [ $this, 'set_user_preferences' ],
+			'get_user_preferences' => [ $this, 'get_user_preferences' ],
+		] );
+
+		$this->mock_editor_visit();
+
+		$this->assertTrue( $this->checklist_module->is_preference_switch_on() );
+	}
+
+	public function test__custom_kit_installed() {
+		parent::setup();
```

Comment:
Why do you run `parent::setup();` in every method? Can you not have a setup method in this class?

Reply:
I want that executed before each test
Does the `setup` method run before each test?